docker/k8s/Dockerfile

```diff
@@ -13,10 +13,11 @@
 # limitations under the License.
 
 ARG VT_BASE_VER=latest
+ARG DEBIAN_VER=buster-slim
```

Comment:
We're doing vitess latest, I wonder if it's worth using Debian stable (buster is currently oldstable)?

Reply:
Done